frontend/public/kubevirt/components/vm/vm-detail.jsx

```diff
@@ -30,31 +32,51 @@ import { Disk } from '../disk';
 import { NodeLink, LoadingInline } from '../../../components/utils';
 import { menuActions } from './menu-actions';
 
-const VmiEvents = ({obj: vm}) => {
+const VmEvents = ({ obj: vm }) => {
   const vmi = {
     kind: VirtualMachineInstanceModel.kind,
     metadata: {
-      name: vm.metadata.name,
-      namespace: vm.metadata.namespace,
+      name: getName(vm),
+      namespace: getNamespace(vm),
     },
   };
-  return <ResourceEventStream obj={vmi} />;
+  const launcherPod = {
+    customFilter: ({ kind, namespace, name }) =>
+      kind === PodModel.kind && namespace === getNamespace(vm) && name.startsWith(`${VIRT_LAUNCHER_POD_PREFIX}${getName(vm)}-`),
+  };
+  const importerPod = {
+    customFilter: ({ kind, namespace, name }) => {
+      // importer pod example importer-<diskName>-<vmname>-<generatedId>
+      // note: diskName may contain '-' which means pod name should have at least 4 parts
+      const podNameSplit = name.split('-');
```

Comment:
it would be safer to use  'cdi.kubevirt.io' labels and reuse `findImporterPods(data, vm)`. Maybe refactor it to something like `isVmImporterPod(pod, vm)` ?

Reply:
I can reuse logic in findImporterPods because Events give you just some small portion of object that generated the event. Basically only name, namespace and kind is available (object that generated the event may no longer exist, eg pod is already deleted etc).

Reply:
I see, that is unfortunate